extension/src/popup/helpers/navigateTo.ts

```diff
@@ -0,0 +1,3 @@
+import { history } from "popup/constants/history";
+
+export const navigateTo = (pathname: string) => history.push({ pathname });
```

Comment:
I really wanted this to accept pathname as type `ROUTES` but we have some instances where we nav using `${ROUTES}?${queryParam}` which ruined my well laid plans

Doesn't appear to be a way to create a type to handle both

Reply:
What if `navigateTo` took a second optional param `queryParams`?

Reply:
Or maybe `route | LocationObject`?